build.xml

```diff
@@ -61,6 +61,32 @@
    </javadoc>
   </target>
 
+  <target name="test" description="Build and run the JUnit test suite.">
```

Comment:
Why not to set this target depends on 'dist' target? This will reuse compile task from dist...

Reply:
Because we don't need the entire build. As our test suite grows, I intend to refactor the tests to be built on a per-project basis anyway.